client/src/components/admin/AdminNews.vue

```diff
@@ -1,6 +1,31 @@
 <template>
   <div id="adminNewsPanel" class="py-3">
-    <div class="table-responsive">
+    <div>
+      <h5>Newsletter</h5>
+      <div class="form-check m-1">
```

Comment:
A new tab would be more logical, as this has nothing to do with the news on /home

Reply:
Kann ich gerne tun. Hatte das auch zuerst im Sinne. Fand es dann aber doof für zwei Elemente einen eigenen Tab zu bauen.

Reply:
Das Wort Hammer ist auch in Hammerhai. Den kann ich aber trotzdem nicht im OBI kaufen😂

Reply:
Der Vergleich hinkt aber auch gewaltig mein Lieber 😜

Reply:
Ne der schwimmt

Reply:
Der Hai ja. Der Hammer nicht.